### PR TITLE
Test mobile menu(Pair Programming with kirti arora)

### PR DESCRIPTION
**Pull request by HtetWaiYan7191**
### Add mobile menu for Portfolio

1. Add new menu-navbar div in html
2. Style menu-navbar 
3. Connect with Javascript to work

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -19,6 +23,75 @@
...
+    width: 100%;
+    height: 100%;
+    top: 0;
+    background-color: rgba(96, 112, 255, 0.9);
+    z-index: 999;
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+    padding: 20px;
+  }
```
Comment (AlphaNtihinduka) [inline style.css]: - [x] Please ensure to have a blurred background color. The app contents should not be clear. This is to ensure that we stick to the Figma template design as much as possible and also have a professional portfolio design

Your design 😎 
image

Figma design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team🙋‍♂️
Good job working on your mobile menu. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement a few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- The closing button works perfectly ✔️ 
- The mobile menu disappears when any option is clicked ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: style.css (hunk 1 above)
Comment (HtetWaiYan7191) [reply style.css]: Thank you very much for clear instructions 😁 🙇

**Review by gemmen29 — APPROVED** [review]
Hi @HtetWaiYan7191, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**